receiver/http.go

```diff
@@ -128,19 +132,26 @@ func (htt *HTTP) Start() error {
 		htt.auth = false
 	}
 	for idx, h := range htt.Handlers {
-		log.Printf("Adding handler for %v", idx)
+		log.WithField("handler", idx).Debug("Adding handler")
 		serveMux.Handle(idx, receiver{Handler: h.H, settings: htt})
 	}
-	if htt.Address == "" {
-		log.Printf("HTTP: No listen-address specified. Using go default (probably :http or :https?)")
-	}
+
 	server.Addr = htt.Address
 	if htt.Certfile != "" {
-		log.Printf("Starting http receiver with TLS at %s", htt.Address)
+		log.WithField("address", htt.Address).Info("Starting http receiver with TLS")
 		log.Fatal(server.ListenAndServeTLS(htt.Certfile, htt.Keyfile))
 	} else {
-		log.Printf("Starting INSECURE http receiver (no TLS) at %s", htt.Address)
+		log.WithField("address", htt.Address).Debug("Starting INSECURE http receiver (no TLS)")
```

Comment:
Arguably, I would invert this: I think starting in insecure mode is more important than starting with TLS :D But that's just the way I like _my_ bikeshed to be painted.